README.md

```diff
@@ -112,4 +112,4 @@ Implementation notes:
 <h5>Localization</h5>
 
 - The component map attempts to load a bundle named "model_translation" (e.g. "model_translation.properties", "model_translation_fr.properties", etc.) in the app and look
-	for keys matching enum-class literal names (e.g. "model.Task$TaskType.Normal=Normal (en)")
\ No newline at end of file
+	for keys matching enum-class literal names (e.g. "model.Task$TaskType.Normal=Normal (en)")
```

Comment:
Not sure why this popped in here. Probably the file/line ending difference thing, as I did a commit via the web GH interface.